pkg/stub/config.go

```diff
@@ -174,21 +174,21 @@ func (h *Handler) buildSkipFilters(opcfg *v1.Config) {
 	defer h.mapsMutex.Unlock()
 	newStreamMap := make(map[string]bool)
 	newTempMap := make(map[string]bool)
-	for _, st := range opcfg.Spec.SkippedTemplates {
+	for _, st := range opcfg.Status.SkippedTemplates {
 		newTempMap[st] = true
 	}
-	for _, si := range opcfg.Spec.SkippedImagestreams {
+	for _, si := range opcfg.Status.SkippedImagestreams {
 		newStreamMap[si] = true
 	}
 	h.skippedImagestreams = newStreamMap
 	h.skippedTemplates = newTempMap
 
 }
 
-func (h *Handler) buildFileMaps(cfg *v1.Config) error {
+func (h *Handler) buildFileMaps(cfg *v1.Config, cfgChange bool) error {
```

Comment:
maybe rename this to forceRebuild ?  That seems to be its intent.

Reply:
Yeah I went with the "why" vs. "intent" etc.

But there may be new reasons to force in the future ... I'll change

Reply:
part of next push